pset-iterators.js

```diff
@@ -160,7 +176,11 @@ Expected Result Shape: A string representing the full name of the president that
 console.log('Problem 6:')
 
 // Add your code below this line
+const firstRepublican = presidents.filter(function (president) {
```

Comment:
@deedeemedina your test condition inside your callback is correct, however, you want to use `.find()` instead of `.filter()` here since the problem asked for the *first* republican and not all republicans